claim-check/code-samples/sample-4/client-producer/Program.cs

```diff
@@ -44,28 +45,13 @@ private static async Task<PayloadDetails> ProcessAsync()
             string storageConnectionString = ConfigurationManager.AppSettings["STORAGE_CONNECTION_STRING"];
             
             // Check whether the connection string can be parsed.
-            if (CloudStorageAccount.TryParse(storageConnectionString, out var storageAccount))
+            if (storageConnectionString!=null & storageConnectionString !="")
```

Comment:
NIT: `if (!string.IsNullOrEmpty(storageConnectionString))` might be slightly more idiomatic.